AmericanMedicalAssociation/joe-style-guide
# EWLJ-410 SG - Multiple popup references to articles

## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Jira Ticket**

- [EWLJ-410: SG | Multiple popup references to articles](https://issues.ama-assn.org/browse/EWLJ-410)


## Description
References that have multiple <sup> numbers in article should connect to the relevant footnotes


## To Test

- switch JOE SG2 branch to `feature/EWLJ-410-Multiple-popups`
- run `gulp serve`
- visit http://localhost:3000/?p=pages-article
- hover over the sup tags with multiple numbers
- observe different reference popups when the mouse is over the number
- visit http://ama-joe.local/article/innovating-nanoethics/2019-04
- hover over the sup tags with multiple numbers
- observe multiple references pop up when the mouse is over the number
- observe the popup text is the same as the references below the article

## Visual Regressions

n/a


## Relevant Screenshots/GIFs

<img width="1085" alt="Screen Shot 2019-05-06 at 4 58 56 PM" src="https://user-images.githubusercontent.com/2271747/57257839-41665280-7020-11e9-92ee-fe6a0dc00162.png">


## Remaining Tasks

Remaining tasks?


## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?
